src/coreComponents/finiteElement/elementFormulations/H1_Tetrahedron_Lagrange1_Gauss.hpp

```diff
@@ -38,15 +38,16 @@ namespace finiteElement
  *           +                               Node   r   s   t
  *           |\\_                            =====  ==  ==  ==
  *           ||  \_                          0      0   0   0
- *           | \   \_           t            1      1   0   1
+ *           | \   \_           t            1      1   0   0
  *           |  +__  \_         |   s        2      0   1   0
  *           | /2  \__ \_       |  /         3      0   0   1
  *           |/       \__\      | /          =====  ==  ==  ==
  *           +------------+     *------r
  *          0              1
  *
  */
-class H1_Tetrahedron_Lagrange1_Gauss1 final : public FiniteElementBase
+template< int NUM_Q_POINTS >
```

Comment:
```suggestion
template< typename NUM_Q_POINTS >
```
and then `NUM_Q_POINTS::value` to use it and`H1_Tetrahedron_Lagrange1_Gauss< std::integral_constant< int, 1 > >`